src/table.js

```diff
@@ -538,6 +538,8 @@ export default class Table {
         cell = this.getRow(rowIndex).appendChild(cellElem);
       }
 
+      this.setCellBackgroundColor(rowIndex, columnIndex > 0 ? columnIndex : numberOfColumns + 1, this.config.defaultBackgroundColor);
```

Comment:
Should this line:
```
columnIndex > 0 ? columnIndex : numberOfColumns + 1
```
be
```
columnIndex > 0 ? columnIndex : numberOfColumns /* without plus 1 */
```
instead?

Assuming that index is 1-based, the highest column index would be equal to number of columns, no?